pipelines/deployment.yml

```diff
@@ -1129,7 +1129,7 @@ jobs:
       environment_variables:
         <<: *ci_security_user
         <<: *ci_service_endpoints_for_python
-        EQ_URL: 'https://eq-test/session?token='
```

Comment:
Is there going to be a change in frontstage for this URL bit as well?

Reply:
@liamtoozer no, my mistake!